velox/docs/functions/presto/conversion.rst

```diff
@@ -0,0 +1,269 @@
+====================
+Conversion Functions
+====================
+
+During expression evaluations, Velox doesn't implicitly convert expression
+arguments to the correct types. If such a conversion is necessary, two provided
+conversion functions can be used explicitly cast values to a particular type.
+
+Conversion Functions
+--------------------
+
+.. function:: cast(value AS type) -> type
+
+    Explicitly cast a value as a type. This can be used to cast a varchar to a
+    numeric value type and vice versa.
+
+.. function:: try_cast(value AS type) -> type
+
+    Like :func:`cast`, but returns null if the cast fails. ``try_cast(x AS type)``
+    is different from ``try(cast(x AS type))`` in that when there is an error
+    during the evaluation of ``x``, ``try_cast`` throws, while ``try(cast())``
```

Comment:
This description seems incorrect. It reads that try_cast hides exceptions that happen when evaluating 'x'. However, try_cast should not affect evaluation of 'x', rather hide exceptions that occur while casting x. CC: @laithsakka

Reply:
I think I meant the same thing. What about I rewrite it to ``try_cast`` throws when there is an error during the evaluation of ``x``, while ``try(cast())`` suppresses the error and turns it to NULL in the result vector?

Reply:
I see. I'm afraid other readers may be as confused as I was. Maybe provide an example?